packages/editor-ui/src/composables/useUserHelpers.ts

```diff
@@ -0,0 +1,36 @@
+import type { RouteLocation } from 'vue-router';
+import { useRouter, useRoute } from 'vue-router';
+import { hasPermission } from '@/utils/rbac/permissions';
+import type { PermissionTypeOptions } from '@/types/rbac';
+
+export function useUserHelpers() {
+	const router = useRouter();
+	const route = useRoute();
```

Comment:
To follow the pattern we have in the other composables, can you add the router and route as composable arguments? See `usePushConnection`.